include/retdec/common/function.h

```diff
@@ -37,7 +37,8 @@ class Function : public retdec::common::AddressRange
 	public:
 		enum eLinkType
 		{
-			USER_DEFINED = 0,
+			DECOMPILER_DEFINED,
+			USER_DEFINED,
```

Comment:
Consider adding a comment describing the difference between `DECOMPILER_DEFINED` and `USER_DEFINED`. We know the difference now, but in the future, we or anybody else may wonder about that.

Reply:
Yes, you are right. I think it would be good idea, too.